hamlet/settings/aws.py

```diff
@@ -0,0 +1,124 @@
+import os
+import sys
+
+from .base import *  # noqa
+from gensim.models.doc2vec import Doc2Vec
+
+# CHECK FOR EC2 TO ADD IP TO ALLOWED HOSTS
+#-----------------------------------------------------------------------------
+def is_ec2_linux():
+    """Detect if we are running on an EC2 Linux Instance
+        See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/identify_ec2_instances.html
+    """
+    if os.path.isfile("/sys/hypervisor/uuid"):
+        with open("/sys/hypervisor/uuid") as f:
+            uuid = f.read()
+            return uuid.startswith("ec2")
+    return False
+
+def get_linux_ec2_private_ip():
+    """Get the private IP Address of the machine if running on an EC2 linux server"""
+    try:
+        from urllib2 import urlopen
+    except:
+        from urllib.request import urlopen
+    if not is_ec2_linux():
+        return None
+    try:
+        response = urlopen('http://169.254.169.254/latest/meta-data/local-ipv4')
+        return response.read()
+    except:
+        return None
+    finally:
+        if response:
+           response.close()
+#--------------------------------------------------------------------------
+
+
+# DATABASE CONFIGURATION
+# -----------------------------------------------------------------------------
+
+DATABASES = {  
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.environ['RDS_DB_NAME'],
+        'USER': os.environ['RDS_USERNAME'],
+        'PASSWORD': os.environ['RDS_PASSWORD'],
+        'HOST': os.environ['RDS_HOSTNAME'],
+        'PORT': os.environ['RDS_PORT'],
+    }
+}
+
+# GENERAL CONFIGURATION
+# -----------------------------------------------------------------------------
+#SET SECRET KEY IN .ebextensions config
```

Comment:
Helpful!

Reply:
(Do we actually do that? How/where?)

Reply:
Nope, plan is no passes/etc in .ebextensions now will change